### ISSUE TITLE
All demo examples are broken

### ISSUE THREAD

**Issue by adjourn** [ianstormtaylor/slate#2180]
#### Do you want to request a _feature_ or report a _bug_?

Bug

#### What's the current behavior?

All the examples at [https://www.slatejs.org](https://www.slatejs.org) are broken. 

Caret goes to the start of document on key press, writes that one letter and then disappears (no selection).

I get the following error:

```
immutable.js:3690

Uncaught Error: Cannot set unknown key "marks" on a
    at a.nt.set (immutable.js:3690)
    at Object.onSelect (after.js:682)
    at a.value (stack.js:114)
    at editor.js:298
    at e.value (change.js:132)
    at a.change (editor.js:235)
    at a.onEvent (editor.js:297)
    at Object.a.(/anonymous function) [as onSelect] (https://www.slatejs.org/main-a107a9db9b049bba176e.js:1:178444)
    at HTMLDocument.<anonymous> (content.js:384)
    at k (debounce.js:95)
```

Tried Chrome, Firefox and Edge, same story. Windows 10. Examples worked last time (few days ago).

#### What's the expected behavior?

Examples work as intended.

**Comment by adjourn** [comment]
Fixed by #2181.

**Comment by ianstormtaylor** [comment]
Thanks for reporting this @adjourn, the examples should be fixed now.
